portfolio/src/main/webapp/More.html

```diff
@@ -21,13 +22,21 @@ <h1>Emma Wenzel</h1>
         <h3>Click here to get a random fact:</h3>
         <button onclick="addRandomFact()">Facts!</button>
         <div id="fact-container"></div>
-          <br>
-          <h3>This is a picture of a seagull.</h3>
-          <p> Seagulls are my second favorite
-              type of bird. This particular seagull is <i>french</i>; we'll call him
-              Louis. I found Louis at Le Mont-Saint-Michel. He posed for my picture.
-          </p>
-          <img src="/images/IMG_5916.JPG" alt="seagull?" width = "300">
+        <br>
+        <h3>This is a picture of a seagull.</h3>
+        <p> Seagulls are my second favorite
+            type of bird. This particular seagull is <i>french</i>; we'll call him
+            Louis. I found Louis at Le Mont-Saint-Michel. He posed for my picture.
+        </p>
+        <img src="/images/IMG_5916.JPG" alt="seagull?" width = "300">
+        <br>
+        <br>
+        <h3>Which Bird is the Best?</h3>
+        <p>To solve this argument we looked at the number of bird species in Colorado
+            by bird. According to the chart below, it looks like ducks are the 
```

Comment:
"number of bird species in Colorado by bird." I'm not sure what this means. Do you mean "number of birds by species", or something else? The title of the table says "bird by species count", which seems kinda different.

Okay, so I checked wikipedia. Are you trying to say that there are 42 species of duck in Colorado?

This is important information. The people must know. And it's important to communicate effectively.